frontend/src/metabase/query_builder/components/view/ViewHeader/components/SavedQuestionLeftSide/SavedQuestionLeftSide.tsx

```diff
@@ -92,13 +92,15 @@ export function SavedQuestionLeftSide(props) {
       </ViewHeaderMainLeftContentContainer>
       {isAdditionalInfoVisible && (
         <ViewHeaderLeftSubHeading>
-          {QuestionDataSource.shouldRender(props) && !isModelOrMetric && (
-            <StyledQuestionDataSource
-              question={question}
-              isObjectDetail={isObjectDetail}
-              subHead
-            />
-          )}
+          {QuestionDataSource.shouldRender({ question, isObjectDetail }) &&
+            !isModelOrMetric && (
+              <StyledQuestionDataSource
+                question={question}
+                isObjectDetail={isObjectDetail}
+                originalQuestion={undefined} // can be removed, needed for typings
```

Comment:
as this type is defined as `originalQuestion?:`, I think it should be fine to omit this - it should fallback to `undefined` and not result in a type error.

Reply:
ah, I didn't notice that. we can leave this as-is then.